NDB.Covid19/NDB.Covid19.Droid/Utils/LocalNotificationsManager.cs

```diff
@@ -67,8 +89,22 @@ public void GenerateLocalNotification(NotificationViewModel notificationViewMode
         
         public Notification CreateNotification(NotificationViewModel notificationViewModel)
         {
+            string channelId;
+            switch (notificationViewModel.Type)
+            {
+                case NotificationsEnum.NewMessageReceived:
+                    channelId = _exposureChannelId;
+                    break;
+                case NotificationsEnum.BackgroundFetch:
+                    channelId = _backgroundFetchChannelId;
+                    break;
+                default:
+                    channelId = _permissionsChannelId;
+                    break;
+            }
+
```

Comment:
You need also to set priority for notification prior Android O because this will work correctly only for androids that support channels

Reply:
Corresponding priorities to importance level you can find here https://developer.android.com/training/notify-user/channels

Reply:
done